.travis.yml

```diff
@@ -17,6 +19,7 @@ cache:
     - $HOME/.composer/cache/files
 
 before_install:
+  - if [ "$DEPENDENCIES" = "beta" ]; then composer config minimum-stability beta; fi;
```

Comment:
General question: what is better? require `^4.0@dev` and doing a minimum-stability: dev here, or going explicit on beta to catch releases only?

Reply:
It's a great question :). I chose to do beta because I don't think it makes sense to always test against the *latest* (effectively `dev-master`) version of Symfony. We don't really care of the bundle is compatible with a new Symfony version until it becomes beta at least. So I think this is the best config.

Reply:
But we would have to change twice: 
- from beta to RC
- from RC to stable
having dev for that period of try and fail would cause only one change then: from dev -> stable.
But i think the point of having released versions only sounds valid.

Reply:
I don't think we would need to change this again. This is the *minimum* stability, so I think when 4.0.0 stable is released, this line would cause 4.0.0 to be downloaded. This *should* say "use the latest beta, RC or stable version".